site/sku.routes.js

```diff
@@ -1,6 +1,5 @@
 const fs = require('fs');
 const path = require('path');
-const flatten = require('lodash/flatten');
```

Comment:
Can this dependency be removed now?

Reply:
This is the full `lodash` dependency and we still use other functions in other places